package/scripts/common/quarto

```diff
@@ -2,66 +2,78 @@
 
 # Determine the path to this script (we'll use this to figure out relative positions of other files)
 SOURCE="${BASH_SOURCE[0]}"
-while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+if [ -h "$SOURCE" ]; then
```

Comment:
I changed this because I found that this misbehaved on conda, where there is no symlink at all